cmd/resolver-cli/cmd/resolve.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/paxthemax/resolver/pkg/resolver"
+	"github.com/spf13/cobra"
+)
+
+// TODO: read this in from config.
+const defaultENSEndpoint = "https://cloudflare-eth.com"
+
+func (c *command) initResolveCmd() {
+	c.root.AddCommand(&cobra.Command{
+		Use:   "resolve",
+		Short: "Resolve a name to a hash",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			// Parse arguments:
+			if len(args) != 1 {
+				return errors.New("Expected single argument: name to resolve")
+			}
+			name := args[0]
+
+			// TODO: handle this in the resolver code
+			if !strings.HasSuffix(name, "eth") {
```

Comment:
ens supports not just `.eth` TLD:
https://medium.com/the-ethereum-name-service/why-ens-doesnt-create-more-tlds-responsible-citizenship-in-the-global-namespace-7e66658fe2b1